File: VB/GridWithFilterPanelButtons/CustomGridControl/CustomGridFilterPanelInfoArgs.vb
Imports DevExpress.XtraGrid.Drawing
Imports System
Imports System.Linq

Namespace GridWithFilterPanelButtons
    Public Class CustomGridFilterPanelInfoArgs
        Inherits GridFilterPanelInfoArgs

        Public viewInfo As CustomGridControlViewInfo

        Public Sub New(ByVal info As CustomGridControlViewInfo)
            MyBase.New()
            viewInfo = info
        End Sub
    End Class
End Namespace

File: CS/GridWithFilterPanelButtons/CustomGridControl/CustomGridFilterPanelInfoArgs.cs
using DevExpress.XtraGrid.Drawing;
using System;
using System.Linq;

namespace GridWithFilterPanelButtons
{
    public class CustomGridFilterPanelInfoArgs : GridFilterPanelInfoArgs
    {
        public CustomGridControlViewInfo viewInfo;

        public CustomGridFilterPanelInfoArgs(CustomGridControlViewInfo info) : base()
        {
            viewInfo = info;
        }
    }
}
